src/components/Button/Button.md

```diff
@@ -109,6 +109,36 @@ Set of Design-approved public buttons.
 <Button.Unstyled backArrowIcon>Button.Unstyled + backArrowIcon</Button.Unstyled>
 ```
 
+#### React Nodes as Button children
+
+```jsx
+  <div style={{ backgroundColor: 'var(--BrandForest)', padding: 20 }}>
```

Comment:
I see what you're doing here and I like the verification. I don't know if you need it in the documentation though. I bet you could convert this to a jest snapshot quite easily and still maintain the verification

Reply:
cool yeah, I was on the fence about it, but figured I'd push it as a kind of POC; I can remove it